src/Components/Users/Users.tsx

```diff
@@ -84,9 +84,9 @@ export default function Users() {
   }
 
   useEffect(() => {
-    const timerId = setTimeout(()=>{
+    const timerId = setTimeout(() => {
```

Comment:
here no changes are made to solve the issue of sending multiple requests on every keystroke issue.